templates/etc/init.d/deb/td-agent

```diff
@@ -202,7 +203,7 @@ do_configtest() {
 RETVAL=0
 case "$1" in
 "start" )
-  echo -n "Starting ${TD_AGENT_NAME}: "
+  echo -n "Starting ${TD_AGENT_VERSION}: "
```

Comment:
Wouldn't it make sense to keep the Agent Name as well? Say change it to 
`echo -n "Starting ${TD_AGENT_NAME}-${TD_AGENT_VERSION}:" `